pl.js

```diff
@@ -0,0 +1,778 @@
+account.banned.hack To konto zostało zbanowane za używanie cheatów
+account.banned.appeal Jeśli chcesz odwołać się od bana, skontaktuj się z:
+account.delete Możesz zgłosić żądanie usunięcia Twojego konta, jeśli nie cofniesz tego żądania w ciągu 14 dni od wysłania go, Twoje konto zostanie trwale usunięte.
+account.delete.request Czy jesteś pewien, że chcesz zgłosić żądanie usunięcia Twojego konta?
+account.delete.cancel Czy jesteś pewien, że chcesz cofnąć żądanie usunięcia Twojego konta?
+spin.free.available DARMOWE KR DOSTĘPNE!
+spin.free.claim Można zakręcić tym kołem raz na godzinę po rozegraniu co najmniej jednego meczu.
+spin.free.disable Te wyskakujące okienka można wyłączyć w ustawieniach
+mod.reset.0 Zmiana zacznie obowiązywać po odrodzeniu.
+mod.reset.1 Obiekty na mapie nie ulegają resetowi
+windows.profile.picture Zmień zdjęcie profilowe
+stats.assists Asysty
+stats.nukes Bomby atomowe
+stats.melee Zabójstwa przy użyciu broni białej
+stats.fistkills Zabójstwa w wyniku pobiciu
+stats.thrownkills Zabójstwa w wyniku rzutu bronią białą
+stats.headshots Trafienia w głowę
+stats.wallbangs Trafienia przez ściany
+stats.sprays Użycia sprejów
+windows.host.default.settings Włączenie domyślnych ustwaień pozwala na przechowywanie postępu graczy oraz najwyższych wyników
+windows.host.default.commands *Komendy hosta są wyłączone w tym trybie: /noclip /god /smite itp
+windows.host.rewards Włączenie nagród KR zmusza serwer do używania domyślnych ustawień i pozwala graczom zdobywać KR z funduszy mapy:
+windows.store.wheels.sub Wypróbuj swoje swoje szczęście i zakręć kołem, aby móc wygrać nowe skiny!
+windows.store.currency * Ceny w USD mogą być przeilczone na Twoją lokalną walutę po wartości wymiany ustalonej przez dostawcę płatności lub bank
+windows.store.creator Pomyśl nad wsparciem swoich ulubionych twórców treści gry (CC) używając ich kodu
+windows.chall.customs Nie można zdobywać wyzwań w niestandardowych meczach
+twitch.link Czy jesteś pewien, że chcesz połączyć konto gry z kontem na Twitchu?
+twitch.unlink Czy jesteś pewien, że chcesz rozłączyć konto gry z kontem na Twitchu?
+windows.clan.leave Czy jesteś pewien, że chcesz opuścić ten klan?
+windows.clan.delete Czy jesteś pewien, że chcesz usunąć ten klan?
+windows.clan.kick Czy jesteś pewien, że chcesz wyrzucić {0} z klanu?
+windows.store.premium.purchase Czy jesteś pewien, że chcesz kupić {0} dni Premium?
+windows.playerlist.ban Czy jesteś pewien, że chcesz wyrzucić {0} z tej gry?
+windows.playerlist.flag Czy jesteś pewien, że chcesz oflagować {0}?
+windows.playerlist.unflag Czy jesteś pewien, że chcesz odflagować {0}?
+windows.playerlist.punish Czy jesteś pewien, że chesz podjąć akcję dotyczącą {0}?
+item.view.open Otwórz przeglądarkę modeli
+item.view.more Więcej informacji
+item.sell.cant Przedmiot nie może zostać sprzedany.
+item.sell.quick Czy na pewno chcesz szybko szprzedać {0}?
+social.profile.stats Statystyki ogólne
+social.profile.class Statystyki klasy
```

Comment:
"Statystyki klas" would be better because that part shows what level you have on each class